e2e/tests/core/04-channel/upgrades_test.go

```diff
@@ -47,15 +48,35 @@ func (s *ChannelTestSuite) TestChannelUpgrade_WithFeeMiddleware_Succeeds() {
 	chainBWallet := s.CreateUserOnChainB(ctx, testvalues.StartingTokenAmount)
 	chainBAddress := chainBWallet.FormattedAddress()
 
+	chainAwalletAmount := ibc.WalletAmount{
+		Address: chainBWallet.FormattedAddress(), // destination address
```

Comment:
note: I find this construction to be a little confusing, labeling a wallet as "chainA" but it's address is for chainB?

Reply:
I renamed it!